core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2923,6 +2925,18 @@ public boolean isQuietingDown() {
         return quietDownInfo != null;
     }
 
+    /**
+     * Returns if the quietingDown is a safe restart.
+     * @since TODO
+     */
+    public boolean isPreparingSafeRestart() {
```

Comment:
No need to make this plugin API.

```suggestion
    @Restricted(NoExternalUse.class)
    public boolean isPreparingSafeRestart() {
```